deliverables/README.md

```diff
@@ -31,38 +36,154 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+They are separated for organization (meaning) and clarity (presentation). 
 
 #### HTML
 What are classes and IDs (and how are they different)?  
+In the CSS, a class selector is a name preceded by a full stop (“.”) and an ID selector is a name preceded by a hash character (“#”).
+Classes are used for multiple items while IDs are used only once.
+
 What are elements?  
+Elements are tags and the content within them.
+
 What are tags?  
+The basic structure of an HTML document includes tags, which surround content and apply meaning to it.
+
 What are attributes?  
+Tags can also have attributes, which are extra bits of information. Attributes appear inside the opening tag and their values sit inside quotation marks. Their format is <tag attribute equals value
+
 What are forms?  
+They notate to take information and send it elsewhere. (i.e., collecting emails and sending it to a mailing list)
+
 What is a div?  
+Div is used to chunk code so information can be attached to it. 
+The difference between span and div is that a span element is in-line and usually used for a small chunk of HTML inside a line (such as inside a paragraph) whereas a div (division) element is block-line (which is basically equivalent to having a line-break before and after it) and used to group larger chunks of code.
 
 What are selectors?  
+Selectors are the names given to styles in internal and external style sheets. Whereas HTML has tags, CSS has selectors. 
```

Comment:
A selector is a string of characters that defines a rule for applying CSS rules to elements.  Specifically it is a pattern against which the browser runs a check for every element, in order to determine whether that rule applies to that element.